### ISSUE TITLE
CSS value “initial” should not be reported as error

### ISSUE THREAD

**Issue by elmimmo** [w3c/epubcheck#1267]
epubcheck reports the CSS `font-size: initial;` as an error.

```
$ epubcheck --locale en -mode exp "$epubdir"
Validating using EPUB version 3.2 rules.
ERROR(CSS-020): …/stylesheet.css(435,2): CSS font selector declaration uses unexpected font-size value "initial".
```
Should it?

Specifications:

- [css3-values](https://www.w3.org/TR/css3-values/#keywords)
- [css3-cascade](https://www.w3.org/TR/css3-cascade/#initial0)

**Comment by rdeltour** [comment]
Definitely looks like a bug, thanks for the report!

**Comment by rdeltour** [comment]
This is no longer reported, since #1393.
